plugin/evm/vm_test.go

```diff
@@ -3016,6 +2964,21 @@ func TestSkipChainConfigCheckCompatible(t *testing.T) {
 	require.NoError(t, reinitVM.Shutdown(context.Background()))
 }
 
+func TestMandatoryUpgrades(t *testing.T) {
+	// use modified genesis to set SubnetEVM timestamp to 1
+	genesisJsonModifiedSubnetEVM := strings.Replace(genesisJSONSubnetEVM, "\"subnetEVMTimestamp\":0", "\"subnetEVMTimestamp\":1", 1)
```

Comment:
This seems like something we could easily break if we changed the string formatting slightly.

Could we alter this format to unmarshal the JSON, update the value, and re-marshal it to pass it in as we do in other places?

Reply:
I see, so the expected behavior is that mandatory upgrades are overridden completely based on the Avalanche network enabled network upgrades.

Reply:
nit: can we adopt the convention of using backtick (``) to quote strings containing `"`?